github/event_types.go

```diff
@@ -37,7 +37,10 @@ type CheckRunEvent struct {
 //
 // GitHub API docs: https://developer.github.com/v3/activity/events/types/#checksuiteevent
 type CheckSuiteEvent struct {
-	CheckSuite *CheckSuite `json:"check_suite,omitempty"`
+	CheckSuite struct {
+		*CheckSuite
+		HeadCommit *Commit `json:"head_commit,omitempty"`
```

Comment:
Instead of doing it this way, let's please add the additional `HeadCommit` field to the existing `CheckSuite` struct.

We have done this in many other locations where a particular field is only populated in a webhook event.

When adding the field to the existing struct, please add a comment (similar to the ones that already exist for other fields like this) that describes how the field is currently only being populated in the case of receiving the struct from a webhook event. You should be able to find examples... but please let me know if you can't and I'll find some for you.

Thank you, @binkkatal!

Also, just FYI... please do not use force commits in PRs to this repo so that reviewers can easily see the changes since the last review. Thanks!